CSS/style.css

```diff
@@ -79,9 +81,214 @@ main {
 }
 
 .social-media-icons li {
-  padding: 0 20px;
+  padding: 1px 20px;
 }
 
 .scroll-down-icon {
-  margin: 64px 0 22px;
+  margin: 64px 1px 22px;
+}
+
+.works-section {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉